communication/templates/robot.html

```diff
@@ -6,19 +6,19 @@
     <meta charset="UTF-8"/>
     <title>Scribble.rs - Join Lobby</title>
     <meta name="title" content="Scribble.rs - Join Lobby">
-    <meta name="description" content="{{.MaxPlayers}} Players - {{.MaxRounds}} Rounds">
+    <meta name="description" content="{{.EditableLobbySettings.MaxPlayers}} Players - {{.MaxRounds}} Rounds">
     <meta property="og:type" content="website">
     <meta property="og:title" content="Scribble.rs - Join Lobby">
-    <meta property="og:description" content="{{.MaxPlayers}} Players - {{.MaxRounds}} Rounds">
+    <meta property="og:description" content="{{.EditableLobbySettings.MaxPlayers}} Players - {{.MaxRounds}} Rounds">
     <meta property="og:image" content="{{.RootPath}}/resources/favicon.png">
     <meta property="twitter:card" content="summary_large_image">
     <meta property="twitter:title" content="Scribble.rs - Join Lobby">
-    <meta property="twitter:description" content="{{.MaxPlayers}} Players - {{.MaxRounds}} Rounds">
+    <meta property="twitter:description" content="{{.EditableLobbySettings.MaxPlayers}} Players - {{.MaxRounds}} Rounds">
```

Comment:
This should not be necessary, as it's an anonymous struct, e.g. the fields are treated as if they are in the struct that `EditableLobbySettings` has been embedded in. Did you get an error here?

Reply:
> This should not be necessary, as it's an anonymous struct, e.g. the fields are treated as if they are in the struct that `EditableLobbySettings` has been embedded in. Did you get an error here?

I get an error on the first try and I realize that it's come from your refactoring struct. :v

Reply:
Removed!